packages/components/src/Enumeration/Header/Header.component.js

```diff
@@ -34,6 +34,7 @@ function getAction(action, index) {
 	}
 	return (
 		<Action
+			id={action.id}
```

Comment:
If I look at the propTypes, I see that `headerDefault` is an array of `<Action>` props. Why not spreading those props in `<Action>` and then add our specific ones ?
Furthermore that's what is done when the action is a dropdown, and that will allow to pass anything we want (I'm thinking about the aria attributes for example)
```
<Action
	{...action}
	key={`${index}-enum-header-action`}
	onClick={onClick}
	tooltipPlacement="bottom" // there was a typo here (btooltipPlacement), let's fix it
	hideLabel
	link
/>
```